hello/views.py

```diff
@@ -15,20 +16,42 @@
 
 def index(request):
     # XXX messages should be a list of messages of the biggest movers
+    '''
+    :: stock_movers ::
+    ticker: stock ticker
+    name: name of the company
+    price: current price
+    change: change in price since open
+    pct_change: percent change since open
+    volume: volume traded
+    '''
     messages = list(Message.objects.filter(focus="MSFT"))
     random.shuffle(messages)
```

Comment:
This will return an error if random isn't imported. Might want to put that import statement back in.